### DISCUSSION TITLE
改主题色后部分色彩不同

### DISCUSSION THREAD

**Discussion by bigbug-dev** [the1812/Bilibili-Evolved#3392] [功能建议 / Ideas]
![image](https://user-images.githubusercontent.com/62877853/171825853-3955f753-b2ad-42c6-a78d-1109bcd1da54.png)

**Comment by bigbug-dev** [comment]
以及夜间模式模式下部分字体
![image](https://user-images.githubusercontent.com/62877853/171830105-0c19a4f2-709f-4646-b1de-8881d5a45908.png)
![image](https://user-images.githubusercontent.com/62877853/171830180-c6f16cc9-ae8e-4558-b897-6dbfd7da649c.png)

**Comment by the1812** [comment]
#2615
